src/styles/Form.module.css

```diff
@@ -0,0 +1,68 @@
+
+.formmaincontainer {
+  display: flex;
+  justify-content: center;
+  background-color: #e8e8dc;
+  width: 100%;
+}
+
+.formcontainer {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin-bottom: 200px;
+}
+
+/* line */
+.line2 {
+  height: 0.125rem;
+  background: #f5f6fa;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+}
+
+.titleinput {
+  width: 40%;
+  height: 1vh;
+  padding: 0.813rem 20rem 0.813rem 1.063rem;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
+}
+
+.authorinput {
+  width: 30%;
+  height: 1vh;
+  padding: 0.813rem 5rem 0.813rem 1.063rem;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
+}
+
+.categoryinput {
+  width: 50%;
+  height: 6vh;
+  padding: 0.375rem 0.938rem;
+  border: 2px solid#e8e8e8;
```

Comment:
Colors are repeating on the CSS files. Kindly follow the DRY principles for HTML&CSS and give color attribute to HTML tags instead of declaring here.

Reply:
Thanks it is fixed